<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgOrderStatus</class>
 <widget class="QDialog" name="dlgOrderStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>970</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>970</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="RaDateSearch">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>日期查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="DateEnd">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="DateStart">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LeSearch">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnCancel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>取消选择当前客户</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnChoseCustomer">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>选择客户</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>全部</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>已完成(已洗完)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>未完成(未洗完)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>已发出</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>未发出</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnOrderNotFinished">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>订单未洗完</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnClothesNotBeSent">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>衣服未发送</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnOrderFinished">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>订单已洗完</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BtnClothesSent">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>衣服已发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="LBCnt">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>显示订单总数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LBCurrentOrderID">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>选中的订单流水号：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LBCurrentCustomerName">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>选中的订单客户姓名：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>700</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
